wordpress-mobile/gutenberg-mobile
# Fix list changing style crash

This PR updates the GB reference to fix an issue when switching list styles on the list block.

This is related to https://github.com/wordpress-mobile/gutenberg-mobile/pull/1778 

To test:
 - Start the demo app
 - Select a list block 
 - Switch the style to ordered list and see that no crash happens.

PR submission checklist:
